service/build.gradle

```
@@ -37,3 +43,15 @@ sonarqube {
         property 'sonar.host.url', 'https://sonarcloud.io'
     }
 }
+
+liquibase {
+    activities {
+        catalog {
+            changeLogFile 'src/main/resources/db/changelog.xml'
+            url 'jdbc:postgresql://127.0.0.1:5432/catalog_db'
```

```suggestion
            url 'jdbc:postgresql://localhost:5432/catalog_db'
```

I think `localhost` is more portable: on MacOS/Linux this is translated to a local socket connection which can be faster than TCP/IP and if you theoretically debug on an IPV6 only system `localhost` should just work.

---

That's good to know!